temba/triggers/models.py

```diff
@@ -455,3 +462,53 @@ def fire(self):
         self.last_triggered = timezone.now()
         self.trigger_count += 1
         self.save()
+
+    @classmethod
+    def catch_nlu_triggers(cls, entity, trigger_type):
+        triggers = Trigger.get_triggers_of_type(entity.org, trigger_type)
+
+        for trigger in triggers:
+            nlu_data = trigger.get_nlu_data()
+
+            consumer = NluApiConsumer.factory(entity.org)
+            if consumer:
+                for bot in nlu_data['bots']:
+                    intent, accurancy, entities = consumer.predict(entity, bot)
+                    accurancy = accurancy * 100
+                    if intent in nlu_data[bot] and accurancy >= nlu_data.get('accurancy'):
+                        extra = {
+                            'intent': intent,
+                            'entities': entities
+                        }
+                        trigger.flow.start([], [entity.contact], start_msg=entity, restart_participants=True, extra=extra)
+                        return True
+
+        return False
+
+    def get_nlu_data(self):
+        nlu_data = json.loads(self.nlu_data)
+        nlu_data['intents_replaced'] = ""
+        nlu_data['intents_splited'] = []
+        nlu_data['bots'] = []
+
+        for intent in nlu_data.get('intent_bot'):
+            info_intent = intent.split('$')
+            nlu_data['bots'].append(info_intent[1])
+            nlu_data['intents_replaced'] += "%s - %s, " % (info_intent[0], info_intent[2])
```

Comment:
Could you explain how are you using this one `intents_replaced `, it's a little bit confused